docs/post_installation/peering.md

```diff
@@ -130,32 +116,24 @@ Example `peering.json`:
 }
 ```
 
-### Autopeering
+## Autopeering
 
-Hornet also supports automatically finding peers through the autopeering module. We recommend to only use autopeering
-if you have at least 4 static peers to minimize service distribution in case your autopeered peers are flaky.
+Hornet also supports automatically finding peers through the _autopeering_ module.  To minimize service distribution in case your autopeered peers are flaky, we recommend to only use autopeering if you have at least 4 static peers.
```

Comment:
```suggestion
Hornet also supports automatically finding peers through the _autopeering_ module. To minimize service distribution in case your autopeered peers are flaky, we recommend to only use autopeering if you have at least 4 static peers.
```